include/globals.h

```diff
@@ -0,0 +1,16 @@
+#ifndef _GLOBALS_H_
```

Comment:
The include guard still is not valid, see https://www.securecoding.cert.org/confluence/display/c/DCL37-C.+Do+not+declare+or+define+a+reserved+identifier

> All identifiers that begin with an underscore and either an uppercase letter or another underscore are always reserved for any use.

Let’s just use `KASSE_GLOBALS_H_`? :)

Reply:
Ah! Thank you for providing that link. Lots of good information, will fix the include guards.

Reply:
To be uniform with the other headers, I'll rename it to `GLOBALS_H_`